core/config/docs/core.toml

```diff
@@ -291,6 +291,12 @@ ForceRedirect = false # Default
 # ListenIP specifies the IP to bind the HTTPS server to
 ListenIP = '0.0.0.0' # Default
 
+# Job Distributor stores the configuration for Job Distributor
+[JobDistributor]
+# DisplayName is a friendly, fully qualified name set by the NOP to clearly identify the node in Job Distributor.
+# It should uniquely and unambiguously represent the node when communicating with Job Distributor maintainers or operators
```

Comment:
+ vice-versa -- i.e. we can use this when communicating with NOPs as well

Reply:
https://github.com/smartcontractkit/chainlink/pull/18606/commits/11e6de03c9968ec47c9200164963d408a2b9f6c8